app/views/gardens/_area.html.haml

```diff
@@ -0,0 +1,15 @@
+- if garden.area
+  %span.area
+    = "#{garden.area} "
+    - if garden.area_unit === 'square metre'
+      %span{:title => 'square metre'}
+        m
+        %sup 2
+    - elsif garden.area_unit === 'square foot'
+      %span{:title => 'square foot'}
+        ft
+        %sup 2
+    - elsif garden.area_unit === 'hectare'
+      %span{:title => 'hectare'} HA
+    - elsif garden.area_unit === 'acre'
+      %span{:title => 'acre'} A
```

Comment:
I wonder if this would be more re-useable as a view helper. Also, instead of a long conditional, I might make a hash that acts as dictionary for these abbreviations, like this:

```
area_abbreviations = {
  'square meter' => 'm<sup>2</sup>',
  # etc
}
```
That way you can just look up the abbreviation with `#{area_abbreviations[garden.area_unit]}`

Just my personal opinion though :)